index.html

```diff
@@ -0,0 +1,104 @@
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+      integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main>
+      <header>
+        <div class="container-nav">
+          <a href="index.html" class="logo-title">Marlyn Mayienga</a>
+          <nav class="nav-bar">
+            <ul>
+              <li class="portfolio">
+                <a href="#portfolio">Portfolio </a>
+              </li>
+              <li>
+                <a href="#">About </a>
+              </li>
+              <li>
+                <a href="#contact">Contact </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+        <div class="hamburger-btn">
+            <i class="fas fa-bars"></i>
+          </div>
+      </header>
+      <section class="introduction">
+        <div class="container">
+        <img class="first-ellipse" src="./assets/ellipse_11.png" alt="first-ellipse">
+        <img class="ellipse_cut" src="./assets/ellipse_cut3.png" alt="ellipse_cut">
+        <img class="second_img" src="./assets/img_twp.png" alt="second_img">
+        <img class="first_img" src="./assets/first_img1.png" alt="first_img">
+          <h1 class="greeting">I’m Merlin <br>
+    </h1>
+          <p class="first-paragraph">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <h2 class="connect">
+            LET’S CONNECT
+          </h2>
+          <div class="social-links">
+            <ul>
+              <li>
+                <a href="https://twitter.com/M_ayienga"
+                  ><i class="fab fa-twitter"></i
+                ></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/marlyn-mayienga/"
+                  ><i class="fab fa-linkedin"></i
+                ></a>
+              </li>
+              <li>
+                <a href="https://www.instagram.com/merl.__/?hl=en"
+                  ><i class="fab fa-medium-m"></i>
+                </a>
+              </li>
+              <li>
+                <a href="https://twitter.com/M_ayienga"
+                  ><i class="fab fa-github"></i>
+                </a>
+              </li>
+              <li>
+                <a href="marlymayienga@gmail.com"
+                  ><i class="fab fa-angellist"></i
+                ></a>
+              </li>
+            </ul>
+        <img class="img_three" src="./assets/img_three.png" alt="">
```

Comment:
- [ ] I can't see your social media icon please try to fix that 
![image](https://user-images.githubusercontent.com/61790587/176030102-8709a551-08a2-4358-ae7c-0a9865e2231a.png)